runtime/src/accounts.rs

```diff
@@ -984,12 +1007,14 @@ mod tests {
         assert_eq!(error_counters.account_not_found, 0);
         assert_eq!(loaded_accounts.len(), 1);
         match &loaded_accounts[0] {
-            Ok((a, l)) => {
+            Ok((a, l, d)) => {
```

Comment:
Not sure what cruel person did this to us, but can you give these names instead of expanding the mess?

Reply:
lol @jackcmay!!  https://github.com/solana-labs/solana/blame/master/runtime/src/accounts.rs#L801